### ISSUE TITLE
Consider adding "sortname" Model properties

### ISSUE THREAD

**Issue by tkem** [mopidy/mopidy#940]
This would make it easier for clients to e.g. present a properly sorted list of artists from search results. Corresponding tags are already present in gstreamer, AFAICS.

**Comment by tkem** [comment]
See also https://discuss.mopidy.com/t/fix-trailing-the-in-library/602/5

**Comment by adamcik** [comment]
+1 for adding this, should be a trivial thing to add to the models, and then a bit of followup in getting MPD to sort of it when appropriate and maybe including it in responses.

**Comment by adamcik** [comment]
```
tagtypes
tagtype: Artist
tagtype: ArtistSort
tagtype: Album
tagtype: AlbumArtist
tagtype: AlbumArtistSort
tagtype: Title
tagtype: Track
tagtype: Name
tagtype: Genre
tagtype: Date
tagtype: Composer
tagtype: Performer
tagtype: Disc
tagtype: MUSICBRAINZ_ARTISTID
tagtype: MUSICBRAINZ_ALBUMID
tagtype: MUSICBRAINZ_ALBUMARTISTID
tagtype: MUSICBRAINZ_TRACKID
OK
```

for a quick overview of the types available in MPD as of 0.18 - i.e. we should make sure to expose these new fields in our MPD impl.

**Comment by east825** [comment]
I'll try to implement it (as part of the sprint session on EuroPython 2015).

**Comment by adamcik** [comment]
Other step that would nice to have for this is updating the `mopidy.audio.utils.convert_tags_to_track` to populate the sortname data in the fields. The field name from GStreamer should be `musicbrainz-sortname`

**Comment by jodal** [comment]
Fixed by #1237.
